malcolm/core/tags.py

```diff
@@ -161,3 +162,8 @@ def method_return_unpacked():
     """
     tag = "method:return:unpacked"
     return tag
+
+def hidden_method():
```

Comment:
```suggestion
def method_hidden():
```
For consistency with the tag text